Makefile

```diff
@@ -266,10 +258,6 @@ set-manifest-pull-policy:
 ## Deploying
 ## --------------------------------------
 
-# Run against the configured Kubernetes cluster in ~/.kube/config
-run: generate fmt vet
-	go run ./main.go
-
```

Comment:
This one is working fine so I think we can keep it.

Reply:
humm, how is it working for you? I get the following error

```
E0907 09:05:09.329242   24327 main.go:178] "setup: problem running manager" err="open /tmp/k8s-webhook-server/serving-certs/tls.crt: no such file or directory"
exit status 1
```
This deploys the webhook as well, and simply running main.go wont work, which was the reason we removed the run phony from CAPM3 as well.

Reply:
Well I mean if you put a certificate there it works fine. Maybe it is not so useful, I just thought that there is nothing "broken" with this since it runs the manager as expected.

Reply:
This used to work without deploying certificate when webhook server was not deployed with manager but separately. Now you need to generate and deploy certificates to make this phony to work, I believe tilt deploy would be much easier. I can add the build phonies as we have in CAPM3 though, those may be useful to check if we can build the go modules.